doc: README.md usage information (#14)
input-output-hk/plu-stan: README.md

| PLU-STAN-02 | Usage of 'unsafeFromBuiltinData' can lead to unexpected behavior | Security    | High      |
| PLU-STAN-03 | Usage of Optional types in on chain code. |           | Security    | Medium    |
| PLU-STAN-04 | Usage of 'eq instance' on script hash, public key hash and payment credential | Security    | Medium    |

## Usage
[[Back to the Table of Contents] ↑](#table-of-contents)
  
### Building & Running

> NOTE: because of cardano specific libraries, we need the following external packages installed: `secp256k1`, `sodium` and `blst`

Reviewer: ```suggestion
NOTE: Because of cardano specific libraries, we need the following external packages installed: `secp256k1`, `sodium` and `blst`. Follow the instructions from the official `cardano-node` install [page](https://developers.cardano.org/docs/operate-a-stake-pool/node-operations/installing-cardano-node/).
```

> We used the ghc 9.6.6 version for development, but other versions supported by the Plinth and Stan should work as well.

Reviewer: ```suggestion
We used ghc version 9.6.6 for development. Other versions supported by Plutus and Stan should work as well. Please report any issues.
```

>   1. to build the project, run:

Reviewer: ```suggestion
  1. To build the project, run:
```

     ```bash
     cabal build
     ```

>   2. to run the tool, inside this project directory, use:

Reviewer: ```suggestion
  2. To run the tool, inside this project directory, use:
```

     ```bash
     cabal run stan
     ```

>   3. to run it outside this project directory, use:

Reviewer: ```suggestion
  3. To run it outside this project directory, use:
```
